src/nat/profiler/calc/calc_runner.py

```
@@ -139,9 +139,9 @@ def validate_config(self) -> None:
         Validate the configuration parameters.
         Raises ValueError if configuration is invalid.
         """
-        # atleast two concurrencies are needed to estimate the GPU count
+        # at least two concurrencies are needed to estimate the GPU count
         if len(self.config.concurrencies) < 2:
-            raise ValueError("Atleast two concurrencies are needed to estimate the GPU count.")
+            raise ValueError("At least two concurrencies are needed to estimate the GPU count.")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)